metricbeat/module/vsphere/resourcepool/_meta/fields.yml

```diff
@@ -0,0 +1,114 @@
+- name: resourcepool
+  type: group
+  release: beta
```

Comment:
The new metricsets are Beta and module as GA.
Is there any other example where we do this. Want to see what type of guidance we usually follow in such scenarios.

Reply:
We follow this approach while adding new metricsets to other modules as well.